Sources/Mantis/CropViewConfig.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+public struct CropViewConfig {
+    public var cropViewMinimumBoxSize: CGFloat = 42
+    public var cropViewPadding: CGFloat = 14
+    public var hotAreaUnit: CGFloat = 32
+    public var cropShapeType: CropShapeType = .rect
+    public var cropVisualEffectType: CropVisualEffectType = .blurDark
+    public var minimumZoomScale: CGFloat = 1
```

Comment:
Is `1` really the lower bound or can we go even further down (e.g. 0.5), if so, I think we should document it.

Reply:
Initially only zoom in is allowed (otherwise the crop box will be bigger than the image), so I think it is reasonable to set minimumZoomScale to 1, although it might be changed later.